x-pack/plugins/apm/public/components/app/correlations/use_failed_transactions_correlations.ts

```diff
@@ -209,7 +226,12 @@ export function useFailedTransactionsCorrelations() {
       );
 
       responseUpdate.fieldStats = stats;
-      setResponse({ ...responseUpdate, loaded: LOADED_DONE, isRunning: false });
+      setResponse({
+        ...responseUpdate,
+        fallbackResult,
+        loaded: LOADED_DONE,
+        isRunning: false,
+      });
```

Comment:
Now that we are no longer using kibana search strategies, would it be possible to use `useFetcher` or is that still blocked by something?

Reply:
As discussed in Slack, since this might be out of scope of the PR, let's table this for now and revisit in the future for a better implementation.